cpp/util/bignum.h

```diff
@@ -0,0 +1,162 @@
+#ifndef CERT_TRANS_UTIL_BIGNUM_H_
+#define CERT_TRANS_UTIL_BIGNUM_H_
+
+#include <glog/logging.h>
+#include <openssl/bn.h>
+
+namespace cert_trans {
+
+
+class BigNum {
+ public:
+  BigNum();
+  explicit BigNum(int64_t w);
+  BigNum(const BigNum& other);
+  ~BigNum();
+
+  BIGNUM* bn();
```

Comment:
Or maybe make it `const` (return value and method), at least? But I have a feeling this is to pass into some OpenSSL API that should have a `const BIGNUM*`, but forgot the `const`, so we're all going to cry...